aop/src/views/AssignmentOfPayment.vue

```diff
@@ -23,6 +24,17 @@ export default {
       showSignOut: false,
     };
   },
+  created() {
+    const self = this;
+    const idle = new IdleJs({
+      idle: 900000, // idle time in ms
+      onIdle() {
+        self.showTimeout = true;
+      },
```

Comment:
If you used an arrow function here, you can get around using the `self` context. No real need for a fix, just more of an FYI.